topics/browser-extension/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Browser extension
+short_description: A browser extension is a small software module for customizing a web browser.
+topic: browser-extension
+related: javascript, typescript
+---
+A browser does typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

Comment:
The extensions themselves aren't cookie management, ad blocking, and the custom scripting and styling of webpages, those are functions performed by the extensions
```suggestion
Web browsers typically have extensions that can do a variety of things, including user interface modification, cookie management, ad blocking, and the custom scripting and styling of web pages.
```